backend/tracim_backend/tests/functional/test_elasticsearch_search.py

```diff
@@ -490,7 +490,9 @@ def test_api___elasticsearch_search_ok__wildcard__facet_only(
 
         web_testapp.authorization = ("Basic", ("test@test.test", "test@test.test"))
         res = web_testapp.get(
-            "/api/advanced_search/content", status=200, params={"search_string": "*", "size": 0}
+            "/api/advanced_search/content",
+            status=200,
+            params={"search_string": "bob riyad", "size": 0},
```

Comment:
For this specific test, it's better to either rename the method (it's not more wildcard) of reset search string to "*" . It's the only place where wildcard is needed and as there is no result there will not been issue with ordering.